shell/platform/fuchsia/flutter/default_session_connection.cc

```diff
@@ -175,6 +181,21 @@ DefaultSessionConnection::DefaultSessionConnection(
     uint64_t max_frames_in_flight,
     fml::TimeDelta vsync_offset)
     : session_wrapper_(session.Bind(), nullptr),
+      inspect_node_(
+          dart_utils::RootInspectNode::CreateRootChild("vsync_stats")),
+      vsyncs_requested_(inspect_node_.CreateUint("VsyncsRequested", 0u)),
+      vsyncs_completed_(inspect_node_.CreateUint("VsyncsCompleted", 0u)),
+      frames_requested_(inspect_node_.CreateUint("FramesRequested", 0u)),
+      frames_presented_(inspect_node_.CreateUint("FramesPresented", 0u)),
+      frames_completed_(inspect_node_.CreateUint("FramesCompleted", 0u)),
```

Comment:
If we change the first two events here to be present_requested, and present_submitted, then let's also change frame_completed to present_completed to finalize the Present2() sequence of events.

Reply:
Done